### DISCUSSION TITLE
Remove Docker CI jobs and scripts/docker

### DISCUSSION THREAD

**Discussion by bryevdv** [bokeh/bokeh#13530] [General]
cc @bokeh/core 

At one point, there was some hope that development and CI might move towards being largely docker-based, and that this might help with some things, e.g. building docs on windows, and isolating a version of chrome for the duration of a dev branch. However, this never came to pass, and the one person who was lightly maintaining "Docker things" has stepped back after some of their other commitments changed. Some of the CI jobs have fallen in to dis-use, and no-one on the core team uses Docker for anything Bokeh-related (or at all), so it's inevitable that other things will start to bit-rot as well. I think for purely practical reasons we must consider jettisoning things we cannot actually maintain, so I'd propose removing everything Docker-based from the repo. 

cc @dkapitan @Karel-van-de-Plassche since you have been interested in Bokeh + Docker in the past. If an interested party is able to actively take over care and maintenance of Docker related things, I might be more inclined to reconsider my own position.

**Comment by dkapitan** [comment]
@bryevdv bit-rot ... I learned a new word today 😄. I concur with your points mentioned above.
